trading-hello-service-api/src/main/scala/com/softwaremill/academy/trading/hello/HelloServiceClient.scala

```diff
@@ -1,6 +1,6 @@
 package com.softwaremill.academy.trading.hello
 
-import cats.effect.{Async, Resource}
+import cats.effect.{ Async, Resource }
```

Comment:
**thought (non-blocking):** I'm not sure about the additional spaces, but let's see how it works 😄